packages/lesswrong/lib/collections/lwevents/views.ts

```diff
@@ -63,7 +63,13 @@ LWEvents.addView("gatherTownUsers", (terms: LWEventsViewTerms) => {
 })
 
 // Index used in manual user-by-IP queries, and in some moderator UI
-ensureIndex(LWEvents, {name:1, "properties.ip":1, createdAt:1, userId:1})
+ensureCustomPgIndex(`
+  CREATE INDEX CONCURRENTLY "manual_idx__LWEvents_properties_ip"
+    ON public."LWEvents" USING gin
+    ((("properties"->>'ip')::TEXT))
+    WITH (fastupdate=True)
```

Comment:
https://www.postgresql.org/docs/15/sql-createindex.html suggests `fastupdate` is set to ON by default

Reply:
This was created using the pgAdmin UI, which put it there, and we discussed and it didn't seem like we needed to remove it.

Reply:
Sure, I guess at worst it's redundant, though now I'm wondering if we have it set to OFF by default or something.